pkg/provider/onepassword/onepassword.go

```diff
@@ -340,14 +342,17 @@ func (provider *ProviderOnePassword) GetSecret(_ context.Context, ref esv1beta1.
 		return nil, err
 	}
 
-	// handle files
-	if item.Category == documentCategory {
-		// default to the first file when ref.Property is empty
-		return provider.getFile(item, ref.Property)
+	// handle fields
+	value, err := provider.getField(item, ref.Property)
+	if err != nil && strings.HasSuffix(err.Error(), "got 0") {
```

Comment:
So this just relies on the fact that if we can't find any labels for the property we'll try files instead.

I would much rather we would be able to define the type of thing we are trying to fetch. Because this is unspecified behaviour. Like, a fallback. Meaning, in order to know how to fetch a file, I would have to know that when I look for a property it must NOT match with any existing labels.

Reply:
As a suggestion we can do the same as we do with keyvault:

`certs/a` for certs `keys/b` for keys  and `c`/ `secrets/c` for default behavior

Reply:
So let's do `field/a` and `file/b`. Do you agree with that ?

Reply:
Yah, that sounds good to me 👍

Reply:
@Skarlso I just pushed the changes and updated the tests